public/sass/pages/_explore.scss

```diff
@@ -120,6 +128,33 @@
   }
 }
 
+@media only screen and (max-width: 1210px) {
+  .time-picker-long-date {
+    .select-button {
+      span {
+        display: none;
+      }
+    }
+  }
+
+  .explore-toolbar.splitted {
+    .explore-toolbar-content-item {
+      .datasource-picker {
+        .ds-picker {
+          min-width: 50px;
+          max-width: 50px;
+
+          .gf-form-select-box__single-value {
```

Comment:
I am not sure I think overriding internal css classes from explore is ideal from a maintainable standpoint. Feels a bit fragile. 

Can this be done via prop option on the on the select component instead?

Reply:
Not sure if the react-select has the prop indicating the split, but I can refactor it to ResponsiveButton.